src/renderer/components/composer/Composer.tsx

```diff
@@ -534,3 +576,21 @@ export function useDraft(
     clearDraft,
   }
 }
+
+
+function RecordingDuration({ seconds }: {
+  seconds: number
+}) {
+  let minutes = 0
+  while (seconds >= 60) {
+    seconds /= 60
+    minutes++
+  }
```

Comment:
depending of what you want to do https://momentjs.com/docs/#/durations/ might help, we depend on momentjs already anyway